react/src/pages/Page404.tsx

```diff
@@ -38,7 +38,7 @@ const Page404 = () => {
           <Button
             size="large"
             type="primary"
-            onClick={() => webuiNavigate('/start')}
+            onClick={() => webuiNavigate('/data')}
```

Comment:
If you want to navigate /data instead of /start when PALI mode, please add redirect it in browserRouter in App.tsx. (ref)